gtfs/spec/en/reference.md

```diff
@@ -29,174 +29,176 @@ This document explains the types of files that comprise a GTFS transit feed and
-|   |  | | * **1** - the station entrance is wheelchair accessible (e.g. an elevator is available to platforms if they are not at-grade)  |  
-|   |  | | * **2** - there exists no accessible path from the entrance to station platforms |  
+|  stop_id | ID | **Required** | Identifies a stop, station, or station entrance/exit. Stops, stations or station entrances/exits are collectively referred to as locations. Multiple routes may use the same stop. | 
+|  stop_code | Text | Optional | Short text or a number that identifies the location for riders. These codes are often used in phone-based transit information systems or printed on signage to make it easier for riders to get information for a particular location. The `stop_code` can be the same as `stop_id` if it is public facing. This field should be left "empty" for locations without a code presented to riders. | 
+|  stop_name | Text | **Required** | Name of a location. Use a name that people will understand in the local and tourist vernacular. | 
+|  stop_desc | Text | Optional | Description of a location that provides useful, quality information. Do not simply duplicate the name of the location. | 
+|  stop_lat | Latitude | **Required** | Latitude of a location. | 
+|  stop_lon | Longitude | **Required** | Longitude of a location. |  
+|  zone_id | ID | **Conditionally Required** | Identifies the fare zone for a stop. This field is required if providing fare information using [fare_rules.txt](#fare_rulestxt), otherwise it is optional. If this `stop_id` represents a station or station entrance/exit, the `zone_id` is ignored. |  
+|  stop_url | URL | Optional | URL of a web page about a particular location. This should be different from the `agency_url` and the `route_url` field values. | 
+|  location_type | Enum | Optional | Indicates whether this `stop_id` represents a stop, station, or station entrance/exit. If `location_type` is omitted, all locations are treated as stops. Valid values are: |  
+|   |  | | `0` or "empty" - Stop. A location where riders board or disembark from a transit vehicle. | 
+|   |  | | `1`- Station. A physical structure or area that contains one or more stop. |  
+|   |  | | `2` - Station Entrance/Exit. A point where riders can enter or exit a station from the street. The record must also specify a parent station by setting `parent_station` equal to the relevant `stop_id`. | 
+|  parent_station | ID referencing `stops.stop_id` | Optional | Identifies the station that a stop or station entrance/exit is part of. This station-stop relationship, called a parent-child relationship, can only exist between a station (as the parent) and a stop or station entrance/exit (as the child). Recall that status as a station, stop or station entrance/exit is defined by `location_type`. If the stop or station identified in this record does not have a parent, `parent_station`="empty". Note that a station can never have a parent and a station entrance/exit must always have a parent. |  
```

Comment:
> ID referencing `stops.stop_id`

I love it! Maybe we could use it everywhere!

Reply:
> a station entrance/exit must always have a parent.

Shall we say conditionnally required then?

Reply:
`stops.stop_id` is a much better notation IMHO than saying "the stop_id field in the stops table", because the former is simply object and field notation and is format independent, where the latter is specific to CSV.  So yes, if Leo is suggest using this notation everywhere I agree.

Reply:
In fact it's already used in most places. But not in stops.zone_id for example.